recipes/vphaser2/meta.yaml

```diff
@@ -1,14 +1,17 @@
 about:
     home: 'https://www.broadinstitute.org/scientific-community/science/projects/viral-genomics/v-phaser-2'
-    license: 'https://www.broadinstitute.org/node/4962'
+    license: 'single user license for academic non-commercial research purposes only'
+    license_file: 'V-Phaser-2.0/license/V_Phaser_2_License_v20130317.pdf'
     summary: 'V-Phaser 2 is a tool to call variants in genetically heterogeneous populations from ultra-deep sequence data'
 package:
   name: "vphaser2"
   version: '2.0'
 source:
-  git_url: https://github.com/broadinstitute/v-phaser2.git #[osx or linux64]
+  url: https://github.com/broadinstitute/v-phaser2/archive/6b5483a8e1a9bf24061a0aa211fee18e6b845668.tar.gz  #[osx or linux64]
```

Comment:
@bgruening Should this perhaps use cargo-port instead?

Reply:
Lets put is this way, we do backup nowadays everything on cargo-port as soon as we merge it. To prevent data loss.